src/common/options.cc

```diff
@@ -3948,6 +3948,12 @@ std::vector<Option> get_global_options() {
     .set_default(255)
     .set_description(""),
 
+    Option("osd_force_recovery_op_priority", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
```

Comment:
I wonder if we can come up with a name that is less likely to be confused with forced_recovery

Reply:
I agree and have renamed it to "osd_kick_recovery_op_priority" since the only caller is the `maybe_kick_recovery `helper.

Reply:
makes sense